src/constants/apiPaths.ts

```diff
@@ -1,6 +1,6 @@
 
 const API_PATHS = {
-  product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+  product: 'https://1ey1lyj9ng.execute-api.eu-west-1.amazonaws.com/dev',
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
```

Comment:
You may update all API paths. This way you'll get some other pages working.
Compare:
https://dszl4pcvftgd3.cloudfront.net/admin/products
https://dyzw59fxpwli2.cloudfront.net/admin/products